packages/catala/catala.0.3.0/opam

```diff
@@ -0,0 +1,51 @@
+opam-version: "2.0"
+synopsis: "Low-level language for tax code specification"
+description: """
+The Catala language is designed to be a low-level target for
+higher-level specification languages for fiscal legislation.
+"""
+maintainer: ["contact@catala-lang.org"]
+authors: ["Denis Merigoux"]
+license: "Apache-2.0"
+homepage: "https://github.com/CatalaLang/catala"
+bug-reports: "https://github.com/CatalaLang/catala/issues"
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "ANSITerminal" {>= "0.8.2"}
+  "sedlex" {>= "2.1"}
+  "menhir" {>= "20200211"}
+  "menhirLib" {>= "20200211"}
+  "unionFind" {>= "20200320"}
+  "bindlib" {>= "5.0.1"}
+  "cmdliner" {>= "1.0.4"}
+  "re" {>= "1.9.0"}
+  "zarith" {>= "1.10"}
+  "zarith_stubs_js" {>= "0.14.0"}
+  "dune" {>= "2.2"}
+  "ocamlgraph" {>= "1.8.8"}
+  "calendar" {>= "2.04"}
+  "visitors" {>= "20200210"}
+  "benchmark" {>= "1.6"}
```

Comment:
Looks like you're missing a couple of dependencies: camomile and js_of_ocaml-ppx

```
#=== ERROR while compiling catala.0.3.0 =======================================#
# context              2.0.8 | linux/x86_64 | ocaml-base-compiler.4.08.1 | pinned(https://github.com/CatalaLang/catala/archive/0.3.0.tar.gz)
# path                 ~/.opam/4.08/.opam-switch/build/catala.0.3.0
# command              ~/.opam/4.08/bin/dune build -p catala -j 31 @install
# exit-code            1
# env-file             ~/.opam/log/catala-15-a78dd5.env
# output-file          ~/.opam/log/catala-15-a78dd5.out
### output ###
#       menhir src/catala/surface/parser_errors.ml
# Read 152 sample input sentences and 152 error messages.
# File "src/catala/dcalc/dune", line 4, characters 39-47:
# 4 |  (libraries bindlib unionFind utils re camomile catala.runtime))
#                                            ^^^^^^^^
# Error: Library "camomile" not found.
# Hint: try:
#   dune external-lib-deps --missing -p catala -j 31 @install
# File "src/catala/dune", line 20, characters 7-22:
# 20 |   (pps js_of_ocaml-ppx))
#             ^^^^^^^^^^^^^^^
# Error: Library "js_of_ocaml-ppx" not found.
# Hint: try:
#   dune external-lib-deps --missing -p catala -j 31 @install
# File "src/french_law/dune", line 16, characters 7-22:
# 16 |   (pps js_of_ocaml-ppx))
#             ^^^^^^^^^^^^^^^
# Error: Library "js_of_ocaml-ppx" not found.
# Hint: try:
#   dune external-lib-deps --missing -p catala -j 31 @install

```

Reply:
Oups indeed! I fixed them in my repo, can I retry `opam publish` or do I have to bump the version number?

Reply:
You can directly modify the opam file in this PR, since the fix is just about the metadata there is no need to re-tag and re-release

Reply:
I have added them manually